apps-rendering/src/components/metadata.tsx

```diff
@@ -122,6 +126,60 @@ const linesDarkStyles = css`
 	${darkModeCss`${from.desktop} {display: block;}`}
 `;
 
+const isLive = (design: Design): boolean => design === Design.LiveBlog;
+
+// This styling function is only temprarily used and will be removed
+// after the liveblog header is completed and ackground colours
+// are added
+const tempraryBackgroundStyle = (format: Format): SerializedStyles => {
+	const themeStyles = getThemeStyles(format.theme);
+	switch (format.design) {
+		case Design.DeadBlog:
+			return css`
+				background-color: ${neutral[93]};
+				${from.desktop} {
+					background-color: ${neutral[97]};
+				}
+				@media (prefers-color-scheme: dark) {
+					background-color: ${neutral[10]};
+				}
+			`;
+		default:
+			return css`
+				background-color: ${themeStyles.liveblogBackground};
+				${from.desktop} {
+					background-color: ${neutral[97]};
+				}
+				@media (prefers-color-scheme: dark) {
+					background-color: ${themeStyles.liveblogDarkBackground};
+					${from.desktop} {
+						background-color: ${neutral[10]};
+					}
+				}
+			`;
+	}
+};
+
+const BlogLines: FC<Item> = (item: Item) => (
+	<div>
```

Comment:
Is the `div` significant? Would a fragment (`<></>`) work?

Reply:
Oh great, I didn't know about the fragment. Just changed it to be fragment, thanks